xla/tsl/platform/default/BUILD

```diff
@@ -343,11 +343,7 @@ cc_library(
         "@tsl//tsl/platform:snappy.h",
     ],
     copts = tsl_copts(),
-    defines = ["TF_USE_SNAPPY"] + select({
-        # TF Additional NUMA defines
-        "//xla/tsl:with_numa_support": ["TENSORFLOW_USE_NUMA"],
-        "//conditions:default": [],
-    }),
+    defines = ["TENSORFLOW_USE_NUMA", "TF_USE_SNAPPY"],
```

Comment:
If `TENSORFLOW_USE_NUMA` is defined in all cases you can just remove it entirely. As far as I can see it's only been used in port.cc, so it shouldn't affect downstream users of this target.